src/components/DataTable/utils.ts

```diff
@@ -1,7 +1,8 @@
 import type { ComponentStyleProp } from '../../types';
 import type { ViewStyle } from 'react-native';
+import type {TDataTableRow} from "./types";
 
-export const keyExtractor = (item: { id: string }) => item.id;
+export const keyExtractor = (item: TDataTableRow) => item;
```

Comment:
This causes typescript error in `defaults.native.ts` and `defaults.ts` files

Reply:
okay bro